payment/src/main/java/org/killbill/billing/payment/core/sm/payments/PaymentOperation.java

```diff
@@ -124,6 +112,9 @@ private OperationException convertToUnknownTransactionStatusAndErroredPaymentSta
                                                                                                 null,
                                                                                                 null);
         paymentStateContext.setPaymentTransactionInfoPlugin(paymentInfoPlugin);
+        if (e instanceof OperationException) {
+            return (OperationException) e;
+        }
```

Comment:
It could happen here:
https://github.com/andrenpaes/killbill/blob/http-error-responses/payment/src/main/java/org/killbill/billing/payment/core/sm/payments/PaymentOperation.java#L86
But, to be honest, even though it's a checked exception, I'm not sure it's ever triggered.

The old code would re-wrap that into another OperationException, so I though it made sense to skip that.